index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Harmanjot Singh</title>
+    <link rel="stylesheet" href="./assets/css/styles.css">
+</head>
+
+<body>
+    <header>
+        <h1><a href="./">Harmanjot Singh</a></h1>
+
+        <!-- Nav bar with anchor links -->
+        <nav class="navbar">
+            <ul>
+                <li>
+                    <a href="#about-me">About me</a>
+                </li>
+                <li>
+                    <a href="#my-work">My work</a>
+                </li>
+                <li>
+                    <a href="#contact-me">Contact me</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- Banner area -->
+    <div class="banner">
+        <h2>My portfolio website</h2>
```

Comment:
Consider changing this title. It's the main focal point of the site when you first visit so I think you would benefit more from having something more personal e.g. `Harmanjot Singh | Professional Portfolio` or something similar.

Reply:
Yeah that's absolutely true, don't know how I missed something like that haha, thank you!